build/run

```diff
@@ -0,0 +1,5 @@
+#!/bin/bash
+
+# Mount the project as a read only filesystem to /app and pass
+# along any arugments that were provided to this script
+docker run -v"$(pwd):/app:ro" backus/rubocop-rspec "$@"
```

Comment:
I agree regarding the long flags (I tried to catch all instances where I could eliminate the opaque shorthand) so good catch! Also, I think there is a `--read-only` flag which I could use instead of doing `:ro`